ui/packages/app/src/components/File/index.ts

```diff
@@ -2,8 +2,16 @@ export { default as Component } from "./File.svelte";
 export const modes = ["static", "dynamic"];
 
 export const document = (config: Record<string, any>) => ({
-	type: "{ name: string; data: string }",
-	description: "file name and base64 data as an object",
+	type: {
+		input_payload: "{ name: string; data: string }",
+		response_object:
+			"{ orig_name: string; name: string, size: number, data: string, is_file: boolean}"
+	},
+	description: {
+		input_payload: "object with file name and base64 data",
+		response_object:
+			"file data as object with filename. `{DEMO URL}/file={name}` hosts the data"
```

Comment:
Same comment as above about `Audio` filepaths. I think we can make this a little clearer.

Reply:
Yes made all changes to audio to file and video too